src/services/TestResultsService.ts

```diff
@@ -586,6 +603,13 @@ export class TestResultsService {
     return testTypeClassification === TEST_TYPE_CLASSIFICATION.ANNUAL_WITH_CERTIFICATE && testResult !== TEST_RESULT.ABANDONED;
   }
 
+  private static isValidTestCodeForExpiryCalculation(testCode: string): boolean {
+    return TEST_CODES_FOR_CALCULATING_EXPIRY.IDS.includes(testCode);
```

Comment:
contemplating merits of moving the .toUpperCase to here - safer then relying on input being correct.

Reply:
I'd suggest keeping all test codes in lower case since we have them like that on the backend and we won't need to do any transformation